.github/workflows/ion-java-performance-regression-detector.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Ion Jave performance regression detector
+
+on:
+  push:
```

Comment:
Do we need to add `on: pull request` too? So we can see the performance changes during code review section as well.

Reply:
> Do we need to add `on: pull request` too? So we can see the performance changes during code review section as well.

That's a good idea, changes will be updated in the following commit